### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by akaysing**
Completed the Avatar component with three size option with their classes
html file is for demo
Please review my code

### CODE REVIEW

**Comment by som749898** [comment]
Looks good but from next time please provide a live preview link or image preview in the comment section

File: Components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.container {
+    display: flex;
+    gap: 0.5rem;
+    justify-content: center;
+    align-items: center;
+    margin-top: 20rem;
+
+}
```
Comment (rajkishancode) [inline Components/Avatar/avatar.css]: remove the blank lines

**Review by rajkishancode — APPROVED** [review]
Rest looks good.

**Review by rohanmathur91 — APPROVED** [review]
LGTM

**Pull request merged**